src/controlCenter/widgets/backlight/backlightUtil.vala

```diff
@@ -0,0 +1,113 @@
+namespace SwayNotificationCenter.Widgets {
+    class BacklightUtil {
+
+        [DBus (name = "org.freedesktop.login1.Session")]
+        interface Login1 : Object {
+            public abstract void set_brightness (string subsystem, string name, uint32 brightness) throws GLib.Error;
```

Comment:
Line limit

Reply:
What do you mean here? :upside_down_face: